.github/workflows/ci.yml

```diff
@@ -22,8 +22,14 @@ jobs:
         id: waitForNetlify
         with:
           site_name: "kuma"
-          max_timeout: 1200
+          max_timeout: 2400
       - name: link checker
+        timeout-minutes: 30
         run: |
           npm install broken-link-checker -g
-          blc $URL -rof --exclude 127.0.0.1 --exclude 192.168 --exclude redis.io  | grep -v '───OK───'
+          if [[ ! `blc $URL/docs/dev -rof --exclude 127.0.0.1 --exclude 192.168 --exclude redis.io > out` ]]; then
+            cat out | grep -v  '───OK───'
+            exit 1
```

Comment:
What am I missing, isn't the first line irrelevant if we always `exit 1`? What went wrong with the original line?

Reply:
the `$?` is the exit code of grep not of the cmd so this was always succeeding

Reply:
You could use $PIPESTATUS[0] to get the exit code of the first command in the pipeline.

Reply:
Although, looks like $PIPESTATUS is pretty bash-specific. Probably best to keep it compatible with other shells.

Reply:
Where was `$?` checked? I thought `pipefail` was set with GH actions.